Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java

```diff
@@ -0,0 +1,86 @@
+package com.dwg_karrier.roys;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * Created by userpc on 2017-01-16.
+ */
+
+public class ListActivity extends AppCompatActivity{
+  ListView lv;
+  ArrayList<item> data;
+  Date fin_time; /* expected finish time */
+  Date cur_time; /* current time*/
+  long duration; /* time duration between current_time and finish time */
+
+  protected void onCreate (Bundle savedInstanceState){
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.list);
+
+    final Intent intent = new Intent(this.getIntent());
+    fin_time = (Date)intent.getSerializableExtra("fin_time");
+    cur_time = (Date)intent.getSerializableExtra("cur_time");
+
+    duration = (fin_time.getTime() - cur_time.getTime()) / 60000;
```

Comment:
What does 60000 mean?
In case it's the 60 seconds, something like `final int minute = 60000;` would be nice.